packages/button/src/Button.tsx

```diff
@@ -394,3 +394,19 @@ Button.propTypes = {
 };
 
 export default Button;
+
+<Button name="cancel" onClick={() => console.log('i')}>
+  this is my button
+</Button>;
+
+{
+  /* <Button /> */
+}
+
+<Button as="div" href="href">
+  hi
+</Button>;
+
+{
+  /* <Box href="string" as="div" />; */
```

Comment:
should move to tests

Reply:
yes these tests were added to box that was an oversight !!